parser/parser.go

```diff
@@ -275,11 +276,13 @@ func (p *Parser) expectPeek(t token.Type) bool {
 	return false
 }
 
-// ErrorContext returns the current line and a pointer to the error position.
+// ErrorLine returns the current line and a pointer to the error position.
 // If prev is true, the error position is relative to the previous token instead of current one.
-func (p *Parser) ErrorContext(prev bool) string {
+func (p *Parser) ErrorLine(prev bool) string {
 	line, errPos := p.l.CurrentLine()
 	if prev {
+		// When the error about the previous tokem, adjust the position accordingly.
```

Comment:
```suggestion
		// When the error about the previous token, adjust the position accordingly.
```

Reply:
fixing in next pr :)


any comments on the harder stuff ... :) ?

Reply:
No, I decided it was too complicated, so I would only review the obvious oddities 🤪 🤡